docs/output.md

```diff
@@ -27,6 +47,247 @@ For further reading and documentation see the [FastQC help](http://www.bioinform
 * `zips/sample_fastqc.zip`
   * zip file containing the FastQC report, tab-delimited data file and plot images
 
+### Quality filtering and adaptor rrimming
```

Comment:
```suggestion
### Quality filtering and adapter trimming
```

Reply:
Couple of quick things I spotted :) I'll have a proper look when you have finished.